src/ts/components/Container.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {Link} from 'react-router';
+import AppBar from 'components/AppBar';
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import Search from 'components/Search';
+
+export default function Container(props) {
+  return (
+     <div>
+      <AppBar />
+      <Header />
+      <Search />
```

Comment:
in particular `AppBar` and `Header` will be displayed conditionally. I guess it's OK to show them by default.